### PR TITLE
Prettier changes

### PR DESCRIPTION
**Pull request by sogame**
* Some rule changes after testing the beta published
* Update a couple of dependencies

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -54,7 +54,6 @@ module.exports = {
     'selector-no-vendor-prefix': true,
     'value-no-vendor-prefix': true,
     'string-quotes': 'single',
-    'selector-attribute-quotes': 'never',
```
Comment (sogame) [inline index.js]: This rule conflicts with Prettier, so removing it.

File: index.js [hunk 2]
```diff
@@ -108,7 +107,6 @@ module.exports = {
     ],
     'scss/double-slash-comment-whitespace-inside': 'always',
     'scss/declaration-nested-properties-no-divided-groups': true,
-    'scss/operator-no-newline-after': true,
```
Comment (sogame) [inline index.js]: This rule conflicts with Prettier, so removing it (at least, was creating issues when tested in Backpack).

File: index.js [hunk 3]
```diff
@@ -124,6 +122,7 @@ module.exports = {
           fill: ['currentColor', 'transparent', 'inherit'],
           stroke: ['currentColor', 'transparent', 'inherit'],
         },
+        disableFix: true,
```
Comment (sogame) [inline index.js]: For `stylelint-declaration-strict-value` to be able to auto-fix issues a fix function must be provided. Disabling autofix (consumers can create their own function if they want).

Comment (olliecurtis) [reply index.js]: Ah yes I had seen this one previously and wasn't sure what the fix was so thank you for finding it!

**Pull request merged**